docs/victoriametrics/changelog/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ See also [LTS releases](https://docs.victoriametrics.com/victoriametrics/lts-rel
 ## tip
 
 * FEATURE: upgrade Go builder from Go1.24.6 to Go1.25. See [Go1.25 release notes](https://tip.golang.org/doc/go1.25).
+* FEATURE: [vmsingle](https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/): add `-relabel.configCheckInterval` and `-streamAggr.configCheckInterval` flags for periodical stream aggregation and relabel configuration check. See [#9590](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/9590).
+* FEATURE: [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/): add `-remoteWrite.configCheckInterval` flag for periodical check of all stream aggregation and relabel check. See [#9590](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/9590).
```

Comment:
```suggestion
* FEATURE: [vmagent](https://docs.victoriametrics.com/victoriametrics/vmagent/): add `-remoteWrite.configCheckInterval`, `-streamAggr.configCheckInterval` and `-relabel.configCheckInterval` flags for periodic config reload. When set, `vmagent` scans the corresponding configs for updates and reloads them automatically. See [#9590](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/9590).
```

Reply:
@makasim there are separate feature items for vmagent and vmsingle